VBA/Office-Shared-VBA/articles/about-autosave.md

```diff
@@ -0,0 +1,120 @@
+---
+title: About AutoSave in Office
+ms.prod: MULTIPLEPRODUCTS
+ms.assetid: about-autosave
+---
+
+
+# About AutoSave in Office
```

Comment:
I think the title should make it clear that this article is about AutoSave + Add-ins/Macros, not AutoSave in general. Something like "AutoSave and Macros and Add-Ins in Office" or "How AutoSave impacts macros and add-ins"